packages/maskbook/src/plugins/VCent/UI/TweetDialog.tsx

```diff
@@ -1,18 +1,17 @@
 import { useEffect, useState } from 'react'
 import { makeStyles, createStyles, Button } from '@material-ui/core'
+import { isDarkTheme } from '../../../utils/theme-tools'
 import * as TweetAPI from '../apis/index'
 import { ETHIcon } from '../Icons/ETH'
-import { VCentIcon } from '../Icons/VCent'
+import { VCentIconLight, VCentIconDark } from '../Icons/VCent'
 import { VALUABLES_VCENT_URL } from '../constants'
 
-const useStyles = makeStyles((theme) => ({
+const useLightTheme = makeStyles((themeLight) => ({
```

Comment:
you do not have to repeat yourself. you can know if the theme is dark or light by `theme.palette.mode === 'dark'`

Reply:
ok thanks. updated `TweetDialog.tsx` to check `theme.palette.mode === 'dark'` on Style color properties.  
See commit 0dbfcc5610e4907c9ae8ca61ca57763c3525e69c